.tekton/daily-or-just-trigger.yaml

```diff
@@ -0,0 +1,108 @@
+  pipelineSpec:
+    params:
+      - name: output-image
+      - name: builder-image
+      - name: dockerfile
+      - name: revision
+      - name: target_branch
+    workspaces:
+      - name: workspace
+    tasks:
+      - name: git-clone
+        when: 
+          - input: "{{ target_branch }}"
+            operator: in
+            values: ["daily"]
+        workspaces:
+          - name: clone-workspace
+            workspace: workspace
+        taskSpec:
+          workspaces:
+            - name: clone-workspace
+          steps:
+            - name: clone-clair-in-ci
+              image: registry.access.redhat.com/ubi9/python-39
+              workingDir: $(workspaces.clone-workspace.path)
+              script: |
+                #!/usr/bin/env bash
+                git clone https://github.com/redhat-appstudio/clair-in-ci-db.git
+      - name: build-image
+        when: 
+          - input: "{{ target_branch }}"
+            operator: in
+            values: ["daily"]
+        workspaces:
+          - name: source
+            workspace: workspace
+        params:
+          - name: IMAGE
+            value: quay.io/redhat-appstudio/clair-in-ci:{{revision}}
+          - name: BUILDER_IMAGE
+            value: $(params.builder-image)
+          - name: DOCKERFILE
+            value: $(workspaces.source.path)/$(params.dockerfile)
+        runAfter:
+          - git-clone
+        taskRef:
+          name: buildah
+          kind: ClusterTask
+      - name: trigger-tests-and-push
+        params:
+          - name: revision
+            value: "{{ revision }}"
+          - name: target_branch
+            value: "{{ target_branch }}"
+        runAfter:
+          - git-clone
+        taskSpec:
+          params:
+            - name: revision
+              type: string
+            - name: target_branch
+              type: string
+          steps:
+            - name: tigger-workflow
+              image: registry.access.redhat.com/ubi9/python-39
+              env:
+                - name: GITHUB_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: integration-github-token
+                      key: integration-github-token
+              script: |
+                #!/usr/bin/env bash
+                if [ "$(params.target_branch)" != "daily" ]; then
+                  curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GITHUB_TOKEN" --request POST --data '{"event_type":"trigger_workflow", "client_payload": {"event": "pull_request", "tag": "$(params.revision)"}}' https://api.github.com/repos/redhat-appstudio/clair-in-ci-db/dispatches
+                else
+                  curl -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GITHUB_TOKEN" --request POST --data '{"event_type":"trigger_workflow", "client_payload": {"event": "daily", "tag": "$(params.revision)"}}' https://api.github.com/repos/redhat-appstudio/clair-in-ci-db/dispatches
```

Comment:
so there is no way how to differentiate between push and pullrequest within PAC?

Reply:
How about:
* `{{source_branch}} == {{target_branch}} == "main"`: push event
* `{{source_branch}} == "daily"`: nightly builds
* anything else pull-request

So we can keep only one trigger?

Reply:
Updated.